.github/workflows/build_idf_examples.yml

```
@@ -11,18 +11,38 @@ jobs:
     strategy:
       matrix:
         idf_ver: ["release-v5.0", "release-v5.1", "release-v5.2", "release-v5.3", "release-v5.4", "latest"]
+        target: ["esp32s2", "esp32s3", "esp32p4"]
+        example:
+          - { name: "USB Device", path: "examples/peripherals/usb/device", manifest_path: "examples/peripherals"}
+          - { name: "Network", path: "examples/network", manifest_path: "examples/network" }
+        exclude: 
+          # Exclude esp32p4 for releases before IDF 5.3 for all runner tags (esp32p4 support starts in IDF 5.3)
```

Copy paste error: we don't have runner tags in this CI Jobs

```suggestion
          # Exclude esp32p4 for releases before IDF 5.3 (esp32p4 support starts in IDF 5.3)
```